workflows/sarek.nf

```diff
@@ -56,7 +63,129 @@ def checkPathParamList = [
 for (param in checkPathParamList) if (param) file(param, checkIfExists: true)
 
 // Set input, can either be from --input or from automatic retrieval in WorkflowSarek.groovy
-input_sample = params.build_only_index ? Channel.empty() : extract_csv(file(WorkflowSarek.retrieveInput(params, log), checkIfExists: true))
+ch_from_samplesheet = params.build_only_index ? Channel.empty() : Channel.fromSamplesheet("input")
```

Comment:
how do you deal with the automatic restart from another step with the csv that are produced by sarek when no input specified?

Reply:
🤔 I don't think I modified this code, but do you have a test dataset for this, so we can be sure?

Reply:
regular `-profile test` or `-profile test_cache`
run it once, and then run it again with --input=false and any --step other than mapping.
so probably --step variant_calling for a quick test